src/main/java/org/cryptomator/cli/CryptomatorCli.java

```diff
@@ -49,6 +49,10 @@ private static void validate(Args args) throws IllegalArgumentException {
 
 		for (String vaultName : args.getVaultNames()) {
 			Path vaultPath = Paths.get(args.getVaultPath(vaultName));
+			if ((args.getVaultPasswordPath(vaultName) != null) && args.getVaultPassword(vaultName) == null)
```

Comment:
please move `{` to the same line. We want to keep a consistent code style and are using automatic formatting rules, that would otherwise lead to unnecessary git diffs